### PR TITLE
New A2A Changes

### PR DESCRIPTION
**Pull request by shail-garmin**
Issue: CA-89143

### CODE REVIEW

File: app/src/main/java/com/fitpay/issuerdemo/MainActivity.java [hunk 1]
```diff
@@ -14,51 +19,105 @@
...
+    Spinner response_type;
+    @BindView(R.id.request)
+    TextView request;
 
     private boolean canSkipWithSuccess;
 
+    private String VISA_AUTH_CODE_KEY = "STEP_UP_AUTH_CODE";
+    private String VISA_RESPONSE_KEY = "STEP_UP_RESPONSE";
+    private String MASTER_CARD_AUTH_CODE_KEY = "TAV";
+    private String MASTER_CARD_RESPONSE_KEY = "issuerMobileAppAuthResponse";
```
Comment (Skyler-Arnold) [inline app/src/main/java/com/fitpay/issuerdemo/MainActivity.java:34]: I'm not familiar with how this app is organized, but is there any way to make these more easily configurable? Not a big deal if that would be a pain, these seem pretty unlikely to change.

**Review by Skyler-Arnold — APPROVED** [review]
I had a very minor comment. Approving, but it would be NTH that change made if it's not too much trouble.

File: app/src/main/java/com/fitpay/issuerdemo/MainActivity.java (hunk 1 above)
Comment (shail-garmin) [reply app/src/main/java/com/fitpay/issuerdemo/MainActivity.java:34]: i guess we have to hardcode on the issuer app side. i can add options on UI side

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -24,13 +45,24 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:hint="password"
-        android:inputType="textPassword"
+        android:text="12345"
```
Comment (tjmunn-fitpay) [inline app/src/main/res/layout/activity_main.xml:48]: Is this the default auth code we should be using when it's the auth_code flow in the issuer app?

**Review by tjmunn-fitpay — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (shail-garmin) [reply app/src/main/res/layout/activity_main.xml:48]: Yes, if you want to fail it then change it to something else.

**Pull request merged**